Symfony/app/Resources/views/base.html.twig

```diff
@@ -42,7 +42,15 @@
                         <a href="#contact">Contact</a>
                     </li>
                     <li>
-                        <a href="#login">Login</a>
+                    {% if is_granted('ROLE_USER') %}
+                        <a href="./logout">Logout</a>
+                    {% endif %}
+                        {% if not is_granted('ROLE_USER') %}
```

Comment:
Hi John, I was trying to stay clear of any else's :) fixed in latest commit to use.